app/services/sandbox_seed_data/school_partnerships.rb

```diff
@@ -15,14 +15,30 @@ def plant
   private
 
     def create_school_partnerships
-      LeadProviderDeliveryPartnership.find_each do |lead_provider_delivery_partnership|
-        school = School.order("RANDOM()").first
-
-        next if school.school_partnerships.map(&:lead_provider_delivery_partnership).map(&:id).include?(lead_provider_delivery_partnership.id)
+      ids = LeadProviderDeliveryPartnership.pluck(:id)
```

Comment:
not sure why we need ids here, can't we just loop through `LeadProviderDeliveryPartnership` as it did already?

Reply:
ah its not needed, will undo